src/runtime.js

```diff
@@ -93,6 +94,9 @@ export function installAmpdocServices(ampdoc) {
   // TODO(dvoytenko, #3742): Split into runtime and ampdoc services.
   installActionServiceForDoc(ampdoc);
   installStandardActionsForDoc(ampdoc);
+  if (isExperimentOn(ampdoc.win, 'amp-lightbox-viewer')) {
```

Comment:
So, if we move the manager, this code will also move into `amp-lightbox-viewer.js` and in core we will simply install amp-lightbox-viewer extension under our experiment.